electron-app/src/components/settings/AccountAssetBalances.vue

```diff
@@ -31,30 +41,36 @@ import { Component, Prop, Vue } from 'vue-property-decorator';
 import CryptoIcon from '@/components/CryptoIcon.vue';
 import { AssetBalances } from '@/model/blockchain-balances';
 import { createNamespacedHelpers } from 'vuex';
+import { Currency } from '@/model/currency';
 
 const { mapGetters } = createNamespacedHelpers('balances');
 const { mapGetters: mapSessionGetters } = createNamespacedHelpers('session');
+const { mapGetters: mapBalancesGetters } = createNamespacedHelpers('balances');
 
 @Component({
   components: { CryptoIcon },
   computed: {
-    ...mapGetters(['accountTokens']),
-    ...mapSessionGetters(['floatingPrecision'])
+    ...mapGetters(['accountAssets']),
+    ...mapSessionGetters(['floatingPrecision', 'currency']),
+    ...mapBalancesGetters(['exchangeRate', 'hasTokens'])
```

Comment:
Is the `hasTokens` used anywhere in this component?  I cannot see it anywhere else in the code or in the properties. If it is unused we can remove it from the mapping.

Reply:
nowhere, copy paste error